nemesis/arena_prep.py

```diff
@@ -172,9 +173,12 @@ def return_to_airship():
 
 
 @battle.utils.speedup_decorator
-def battle_farm_all(ap_cp_limit: int = 255, yuna_attack=True, fayth_cave=True):
+def battle_farm_all(ap_cp_limit: int = 255, yuna_attack=True, fayth_cave=True) -> bool:
     logger.debug(f"Battle Start: {memory.main.get_encounter_id()}")
     FFXC.set_neutral()
+    while memory.main.battle_active():
```

Comment:
I feel like this will result in everyone defending. For the entire battle.

Reply:
ooooohhhh shoot I forgot to put it back. That's exactly what it does, was used to test an intentional game over.